test/fuzzing/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.14
+
+WORKDIR /go
+
+RUN apt-get update && apt-get install -y clang
+
+RUN git clone https://github.com/opencontainers/umoci \
+    $GOPATH/src/github.com/opencontainers/umoci && \
+    cd $GOPATH/src/github.com/opencontainers/umoci && \
```

Comment:
If you move the dockerfile to the root of the repo (and call it `Dockerfile.fuzz` or something) then you don't need to do this clone step -- which would be ideal because then you could run the fuzzer on a locally-modified copy or branch.

Reply:
I just pushed an update. Does that look better to you? If so, I will just amend the documentation.

Reply:
Yup, that looks better (with some minor nits about the ordering of instructions).